governance/articles-of-association.md

```diff
@@ -0,0 +1,52 @@
+# Articles of Association for Pulumiverse
+
+Pulumiverse is a non-profit organization to grow a community to share ideas,
+projects and build Open Source software amongst Pulumi users, developers and
+enthusiasts.
+
+This document regulates this organization; you'll find information about
+becoming a contributor, how and when to vote for board members and how policies
+are changed.
+
+## Contributors
+
+Contributors are the core of Pulumiverse. To become a contributor, all you have
+to do is open an issue(s) or pull request(s). Contributors can earn voting
+rights by either becoming a member of the Open Source Initiative or submitting
+more than ten contributions collectively across Pulumiverse repositories. Voting
+rights are only made valid by being added to the membership roster (feel free to
+open a pull request).
+
+## Community-elected Board
+
+Pulumiverse has a board that consists of five board members. Board members hold
+their role for two years; voting is staggered as such:
+
+We elect board members 1 and 2 in year 1, elected for two years. In year 2, we
+elect board members 3, 4 and 5. We elect board members 3, 4 and 5 for one year
+the first year.
+
+Everyone be can be nominated to become a board member, either by being nominated
+by others or sending your nomination to current board members before the annual
```

Comment:
Should we restrict board member seats for contributors with voting rights?

The obvious issue is: If we want to nominate someone they either have to pay for an Open Source Initiative membership or actively contribute to Pulumiverse. This minimizes the number of potential candidates.

The positive side is that we only get people who are committed to Pulumiverse / Open Source.